src/components/ArtifactCollections.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="artifact-collections">
+    <div class="artifact-collections__filters">
+      <ResultsCount :label="localization.info.artifact" :count="artifactsCount" class="artifact-collection-list__results" />
+      <SearchInput v-model="searchTerm" :placeholder="localization.info.artifactSearch" :label="localization.info.artifactSearch" class="artifact-collection-list__search" />
+      <ArtifactTypeSelect v-model:selected="selectedType" class="artifact-collection-list__type" />
+      <ViewModeButtonGroup class="artifact-collections__view-mode-button-group" />
+    </div>
+
+    <template v-if="artifactsCountSubscription.executed && artifactsCount == 0">
```

Comment:
Right now this doesn't take into account filters - will add that functionality at a later date when the `/counts` endpoint is working